tests/py_test.py

```diff
@@ -92,6 +93,10 @@ def asserter(stmt):
         # taken from the size of monsterdata_python_wire.mon, minus 4
         asserter(size == 348)
         buf, offset = util.RemoveSizePrefix(buf, offset)
+    if file_identifier:
+        asserter(MyGame.Example.Monster.Monster.MonsterBufferHasIdentifier(buf, offset))
```

Comment:
Please also check `GetBufferIdentifier` and `HasBufferIdentifier`.

Reply:
@rw I added this test